libminifi/src/CronDrivenSchedulingAgent.cpp

```diff
@@ -56,8 +56,8 @@ utils::TaskRescheduleInfo CronDrivenSchedulingAgent::run(const std::shared_ptr<c
       } else {
         Bosma::Cron schedule(processor->getCronPeriod());
         result = schedule.cron_to_next(from);
-        last_exec_[uuidStr] = result;
-        schedules_.insert(std::make_pair(uuidStr, schedule));
+        last_exec_[uuid] = result;
+        schedules_.insert(std::make_pair(uuid, schedule));
```

Comment:
It's old code, but it's a map! :)

schedules_[uuid] = schedule;

Reply:
I think they have different semantics, your proposal overwrites if it already exists, the current version does not change the map if there was a same key in it, unclear if that was intentional (probably not) but I wanted to change semantics as little as possible

Reply:
👍